Automattic/wp-calypso
# Flag to close window after Jetpack authorization

#### Changes proposed in this Pull Request

* Enable URL parameter to close the window after Jetpack user authorization if `close_window_after_auth=1` is present in the URL. This enables us to use the Jetpack Connect flow in a popup without using postMessage to send the token back to the opener.
* This is very similar (practically identical) to `close_window_after_login`, except that it closes the window after the authorize step is complete.

#### Testing instructions



* This is designed to work with this [WIP Jetpack PR](https://github.com/Automattic/jetpack/pull/20003)
* You can test it by adding `&close_window_after_auth=1` to the authorize URL generated in Jetpack. Exact way to do this depends on your use case.
* I'm mostly looking to make sure this won't blow anything up. You'll need to trust me that it works and is useful.



Related to https://github.com/Automattic/jetpack/pull/20003
